### PR TITLE
contextual action -> quick fix

### LINKED ISSUE #161532: Rename internal use of contextual actions to quick fix
Instead of 2 terms let's standardize on "quick fix" which makes a lot of sense and aligns with the editor. For anything exported we should make sure it's prefixed with `Terminal`.

### LINKED ISSUE #161602: Avoid using "base" in file names that don't export an abstract class
Whenever I read base I think abstract class, let's change terminalBaseContextualActions.ts to use another word like Builtin

### PR DESCRIPTION
**Pull request by meganrogge**
fix #161602
fix https://github.com/microsoft/vscode/issues/161532

### CODE REVIEW

File: src/vs/workbench/contrib/terminal/browser/terminalInstance.ts [hunk 1]
```diff
@@ -217,7 +217,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	 * Enables opening the contextual actions, if any, that are available
 	 * and registering of command finished listeners
 	 */
-	get contextualActions(): IContextualAction | undefined { return this._contextualActionAddon; }
+	get quickFixActions(): QuickFixAction | undefined { return this._quickFixAddon; }
```
Comment (meganrogge) [inline src/vs/workbench/contrib/terminal/browser/terminalInstance.ts:220]: this plural vs singular is bothering me... maybe should rename `QuickFixAction` to `QuickFixActions` or `QuickFixHandler`?

Comment (Tyriar) [reply src/vs/workbench/contrib/terminal/browser/terminalInstance.ts:220]: Just `quickFix` works fine, that aligns with `statusList`

File: src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts [hunk 2]
```diff
@@ -10,13 +10,13 @@ import { ITerminalCapabilityStore, ITerminalCommand, TerminalCapability } from '
...
 import { IContextMenuService } from 'vs/platform/contextview/browser/contextView';
-import { ICommandAction, ITerminalContextualActionOptions } from 'vs/workbench/contrib/terminal/browser/terminal';
+import { ICommandAction, ITerminalQuickFixActionOptions } from 'vs/workbench/contrib/terminal/browser/terminal';
 import { DecorationSelector, updateLayout } from 'vs/workbench/contrib/terminal/browser/xterm/decorationStyles';
 import { IConfigurationService } from 'vs/platform/configuration/common/configuration';
 import { Terminal } from 'xterm';
 import { IAction } from 'vs/base/common/actions';
 
-export interface IContextualAction {
+export interface QuickFixAction {
```
Comment (Tyriar) [inline src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts:19]: This is missing the `I` prefix, plus I think the replacement should be "contextual action" -> "quick fix", _fix_ is the action. Or "terminal quick fix" when we export anything so annoying imports don't show up when developing (editor) quick fixes

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts (hunk 2 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts:19]: yes agree, nice catch

Comment (meganrogge) [reply src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts:19]: in some cases, `quick fix actions` actually make sense though https://github.com/microsoft/vscode/blob/4167f07e40f41e2d5b313b105ccbd7f972c324a7/src/vs/workbench/contrib/terminal/browser/terminalInstance.ts#L612-L617

**Review by Tyriar — APPROVED** [review]

**Pull request merged**